packages/core/src/connectivity/scp-cf/protocol.ts

```diff
@@ -1,9 +1,10 @@
+/* eslint-disable-next-line */
 export enum Protocol {
   HTTP = 'http',
   HTTPS = 'https'
 }
 
-/* eslint-disable-next-line  @typescript-eslint/no-redeclare */
+/* eslint-disable-next-line */
```

Comment:
Is that better?

Reply:
A second rule now complains here because you shouldn't export 2 things under the same name. But this seemed to be intentional so I though it would be fine to disable eslint for the next row here. Not sure what the best approach is here...

Reply:
What is the other rule? Why wasn't it complaining before? And can this be configured to ignore namespaces?